R/plot-construction.r

```diff
@@ -100,6 +100,13 @@ ggplot_add.data.frame <- function(object, plot, object_name) {
   plot
 }
 #' @export
+ggplot_add.function <- function(object, plot, object_name) {
+  stop(
+    "Don't know how to add ", object_name, " to a plot. Did you mean ",
+    object_name, "()?", call. = FALSE
+  )
```

Comment:
Do we use `abort()`? Or will it be done in #3526?

Reply:
As of now, we're using `stop()`. See e.g. here (purposefully not using permalink):
https://github.com/tidyverse/ggplot2/blob/master/R/plot-construction.r#L89-L92

Reply:
OK.